velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -353,11 +395,12 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
 
   if (readRel.has_virtual_table()) {
     return toVeloxPlan(readRel, outputType);
-
   } else {
-    auto tableScanNode = std::make_shared<core::TableScanNode>(
-        nextPlanNodeId(), outputType, tableHandle, assignments);
-    return tableScanNode;
+    return std::make_shared<core::TableScanNode>(
```

Comment:
Should we assert that there is no emit info? Ditto ValuesNode.

Reply:
Since we are not going to add this to `ValuesNode` or `TableScanNode`, not sure such validation is required. But if we want to avoid calling this by mistake and give an error message in such a case, I guess it could be good to have.

Reply:
It would be good to assert that there is no emit node in case some system adds it "by mistake". Without an assert we may end up silently dropping some information from the original plan during conversion.